src/main/java/net/minestom/server/inventory/EquipmentHandler.java

```diff
@@ -146,8 +139,7 @@ default void setItemInHand(@NotNull Player.Hand hand, @NotNull ItemStack stack)
      * @param slot the equipment to get the item from
      * @return the equipment {@link ItemStack}
      */
-    @NotNull
-    default ItemStack getEquipment(@NotNull EntityEquipmentPacket.Slot slot) {
+    default @NotNull ItemStack getEquipment(@NotNull EntityEquipmentPacket.Slot slot) {
```

Comment:
This should be moved into another enum. There's also a similar enum in `Player` as well that could use a new enum for equipment slots.